packages/cloud-cognitive/src/components/ComboButton/ComboButton.js

```diff
@@ -146,15 +122,27 @@ export let ComboButton = React.forwardRef(
 ComboButton = pkg.checkComponentEnabled(ComboButton, componentName);
 ComboButton.displayName = componentName;
 ComboButton.propTypes = {
+  /** Provide an optional flag to disable entire ComboButton */
+  buttonDisabled: bool,
+
   /** Provide the contents of the `ComboButton` */
   children: node.isRequired,
 
   /** Provide an optional class to be applied to the containing node */
   className: string,
 
-  /** Provide an optional flag to disable the ComboButton */
+  /** Provide an optional flag to disable the ComboButton or a selected ComboButtonItem */
   disabled: bool,
 
+  /** Provide an optional flag to disable a divider between ComboButtonItems */
+  hasDivider: bool,
+
+  /** Provide an optional flag to convert a ComboButtonItem to a delete item */
+  isDelete: bool,
+
+  /** Provide an optional flag to disable selected ComboButtonItem */
+  itemDisabled: bool,
+
```

Comment:
I think these props should all be set on `ComboButtonItem` and not `ComboButton`.

Reply:
I placed these here to make sure they would show up in the story. If there is a better way to make that happen I am all ears!